### PR TITLE
Added contributors list through the git hub api

### PR DESCRIPTION
**Pull request by supunTE**
Added contributors list through the git hub API. for my GCI - task Website: Add and Enhance the "Contributors Section" for the FOSSASIA GCI site by using the Github API ( FOSSASIA ). Please check this. 
https://stedirisuriya.github.io/gci17.fossasia.org/

### CODE REVIEW

**Comment by nzec** [comment]
Squash the commits and resolve the conflicts. Then ping the fossasia/fossasia-gci team.

**Comment by isuruAb** [comment]
@STEdirisuriya  please resolve this conflict. then we can merge this.

**Comment by supunTE** [comment]
OK I resolved all conflicts...

**Comment by abishekvashok** [comment]
@STEdirisuriya again you got conflicts

**Comment by akshatnitd** [comment]
Please resolve the conflicts. I cannot review before all the conflicts are resolved

**Comment by supunTE** [comment]
OK. review please

**Review by isuruAb — APPROVED** [review]

**Comment by ParthS007** [comment]
image
image

**Review by ParthS007 — CHANGES_REQUESTED** [review]
image
image

**Comment by ParthS007** [comment]
@STEdirisuriya Why Mentors and Students are not visible in preview link?

**Comment by ba11b0y** [comment]
@ParthS007 those template tags are rendered during deployment . Please check and update your comment so that it can be merged.

**Review by isuruAb — COMMENTED** [review]
please check the mentors section

File: index.html [hunk 1]
```diff
@@ -1,173 +1,225 @@
...
+
+				</li>
+			</ul>
+		</section>
 
-		            </li>
-		        </ul>
-		    </section><section class="bg-secondary" id="mentors">
+		<section class="bg-secondary" id="mentors">
+<<<<<<< HEAD
```
Comment (chashmeetsingh) [inline index.html]: Please fix the merge conflict. There are many places this needs to be fixed, please take a look.

Thanks!

**Review by chashmeetsingh — CHANGES_REQUESTED** [review]
Please resolve the merge issues.

**Comment by supunTE** [comment]
I finished making web site. no error there now
errfix
errfix2

**Comment by ba11b0y** [comment]
Thanks @STEdirisuriya

**Comment by supunTE** [comment]
please approve my review please

**Comment by supunTE** [comment]
@invinciblycool

**Comment by ParthS007** [comment]
image

**Comment by ParthS007** [comment]
@invinciblycool Please see the screenshot and If contributors have four columns then Students and Mentors should also have 4 columns as Uniformity is necessary. I hope that you now understand what I am trying to say

**Comment by supunTE** [comment]
oh yes

**Comment by ba11b0y** [comment]
Yes, I do understand but the student has enhanced the contributors section
using github api. And that is a design issue which can be improved upon by
opening issues and continued contribution.

**Comment by ParthS007** [comment]
ok @invinciblycool :+1: , its ok but there is empty space in contributors section itself.

**Comment by supunTE** [comment]
That is your browser's version problem. Look at those screen shots. 
noerr
noerr2
noerr3
noerr4

**Pull request merged**